R/plotInteractions.R

```diff
@@ -0,0 +1,185 @@
+#' @title Plot interaction graph
+#'
+#' @description Function to plot directed interaction graphs based on symbolic 
+#' edge-lists and vertex metadata.
+#' The user can specify node, node_label and edge aesthetics using dedicated
+#' arguments. The resulting plot can be further refined with `ggplot2` for node 
+#' styling and `ggraph` for edge-specific customization.
+#'
+#' @param out a data frame representing an edge list with columns \code{"group_by",
```

Comment:
maybe we can add that this is usually the output from testInteractions or countInteractions (maybe link them here if possible)